kview/src/main/java/dev/ikm/komet/kview/mvvm/view/journal/JournalController.java

```diff
@@ -959,10 +961,30 @@ public void loadConceptNavigatorPanel() {
         conceptNavigatorControl.setNavigator(navigator);
         conceptNavigatorControl.setHeader("Concept Header");
         conceptNavigatorControl.setShowTags(false);
-        conceptNavigatorControl.setOnAction(items ->
-                populateWorkspaceWithSelection(items.stream()
-                        .map(item -> item.publicId().asUuidArray())
-                        .toList()));
+        conceptNavigatorControl.setOnAction(action -> switch (action) {
+            // single selection
+            case OPEN_IN_WORKSPACE -> item -> {
+                List<UUID[]> uuids = List.<UUID[]>of(item.publicId().asUuidArray());
+                populateWorkspaceWithSelection(uuids);
+            };
+            case SHOW_RELATED_CONCEPTS -> {
+                // Dummy, for now just add the parents of the selected item as related content:
+                TreeItem<ConceptFacade> selectedItem = conceptNavigatorControl.getSelectionModel().getSelectedItem();
+                if (selectedItem != null) {
+                    conceptNavigatorControl.getNavigator().getParentNids(selectedItem.getValue().nid());
+                    List<ConceptFacade> list = Arrays.stream(conceptNavigatorControl.getNavigator().getParentNids(selectedItem.getValue().nid())).boxed()
+                            .map(nid -> (ConceptFacade) Entity.getFast(nid)).toList();
+                    ((ConceptNavigatorTreeItem) selectedItem).setRelatedConcepts(list);
+                }
+                yield i -> System.out.println("Click! on " + i.description());
```

Comment:
Can we use a logger to output to the console?

Reply:
done